packages/cli/src/lingui-extract.ts

```diff
@@ -103,15 +103,21 @@ type CliOptions = {
   files?: string[]
   clean: boolean
   overwrite: boolean
-  locale: string
+  locale: string[]
   prevFormat: string | null
   watch?: boolean
 }
 
 if (require.main === module) {
   program
     .option("--config <path>", "Path to the config file")
-    .option("--locale <locale>", "Only extract the specified locale")
+    .option(
+      "--locale <locale, [...]>",
+      "Only extract the specified locales",
+      (value) => {
+        return value.split(",").filter(Boolean)
```

Comment:
shouldn't the values be trimmed? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim